backend/workflow_manager/endpoint/database_utils.py

```diff
@@ -410,14 +428,25 @@ def create_table_query(
         for key, val in database_entry.items():
             if key not in TableColumns.PERMANENT_COLUMNS:
                 python_type = type(val)
-                if conn_cls == DBConnectionClass.BIGQUERY:
+                if conn_cls == UnstractDBConnectorClass.BIGQUERY:
```

Comment:
Reduce nesting if possible.

Reply:
Nesting reduced. Each db connector will have a field `sql_to_db_mapping` that will convert value to db column type